enhancements/bare-metal-fulfillment/README.md

```diff
@@ -0,0 +1,360 @@
+---
+title: bare-metal-fulfillment
+authors:
+  - Tzu-Mainn Chen
+creation-date: 2025-09-08
+last-updated: 2025-09-15
+tracking-link:
+  - None
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Bare Metal Fulfillment
+
+## Summary
+
+One key component of Virtual Data Center (VDC) fulfillment is the capability to perform bare metal fulfillment. Bare metal
+fulfillment refers to the process through which a tenant acquires bare metal hosts and configures their networking.
+Afterwards, the tenant will be able to perform selected operations on the host: inventory information, power control, and
+serial console access. Any further operations - for example, image-based provisioning - are outside the scope of the O-SAC
+solution and must be integrated by the cloud provider.
+
+We believe this feature set is generic enough to support a wide range of initial bare metal cluster use cases, including:
+
+* support for an environment where developers have to flexibility to install their own operating systems and software
+* support for alternative clusters such as SLURM
+* support for bare metal configuration during O-SAC cluster fulfillment
+
+In this context, a “tenant” is an organization with admin rights to the requested bare metal resources; at the MOC, a tenant
+would be led by a ColdFront PI or manager.
+
+Although this proposal only covers bare metal fulfillment (and not VMs), we believe that developing this bare metal capability
+independently will allow us to create a thorough and reliable implementation that can later be used as a solid base for
+VDC fulfillment.
+
+## Motivation
+
+Bare metal fulfillment is a fundamental requirement for O-SAC, identified as an explicit need for the MOC. It will also serve
+as a component for other fulfillment workflows, such as VDC and OpenShift cluster fulfillment.
```

Comment:
```suggestion
Bare metal fulfillment is a fundamental requirement for O-SAC. It has been identified as an explicit need for the MOC. It will also serve
as a component for other fulfillment workflows, such as VDC and OpenShift cluster fulfillment.
```